pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,78 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryBasicUserRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "list"),
+					Resources: sets.NewString("projectrequests"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("projects", "subjectaccessreview", "localsubjectaccessreview"),
+				},
+			},
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
```

Comment:
each role will need get on namespaces.  That will allow the project to show up in oc get projects.